pytools/test/test_pytools.py

```diff
@@ -31,7 +31,6 @@
 from typing import FrozenSet
 
 
-@pytest.mark.skipif("sys.version_info < (2, 5)")
```

Comment:
Did this even work? I thought the first argument had to be a bool.

Reply:
I *think* it worked, see e.g. here: https://docs.pytest.org/en/7.1.x/historical-notes.html#string-conditions

Reply:
Oooh! Neat!